src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -0,0 +1,114 @@
+        productService.delete(id);
+    }
+
+    @GetMapping
+    @Operation(summary = "FindAll", description = "Find all products")
+    public List<ProductResponseDto> findAll(@RequestParam(defaultValue = "3") @Parameter(name = "Count", description = "Values number at page (default value is 3)") Integer count,
+                                            @RequestParam(defaultValue = "0") @Parameter(name = "Page", description = "Default first page is 0") Integer page,
+                                            @RequestParam(defaultValue = "id") @Parameter(name = "Sort By", description = "Sort by parameter (default parameter is id)") String sortBy) {
+        List<Sort.Order> orders = sortParserService.orders(sortBy);
+        PageRequest pageRequest = PageRequest.of(page, count, Sort.by(orders));
+        return productService.findAll(pageRequest).stream()
+                .map(dtoDtoMapper::mapToDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/price")
+    @ApiOperation(value = "Get all products between price from-t")
+    public List<ProductResponseDto> findAllPriceBetween(
+            @RequestParam(defaultValue = "0")
+            @ApiParam(value = "default value is 0") BigDecimal from,
+            @RequestParam(defaultValue = "10000")
+            @ApiParam(value = "default value is ‘10000’") BigDecimal to,
+            @RequestParam(defaultValue = "10")
+            @ApiParam(value = "default value is 10") Integer count,
+            @RequestParam(defaultValue = "0")
+            @ApiParam(value = "default value is 0") Integer page,
+            @RequestParam(defaultValue = "id")
+            @ApiParam(value = "default value is id") String sortBy) {
+        List<Sort.Order> orders = sortParserService.orders(sortBy);
+        PageRequest pageRequest = PageRequest.of(page, count, Sort.by(orders));
+        return productService.findAllByPriceBetween(from, to, pageRequest)
+                .stream()
+                .map(dtoDtoMapper::mapToDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/test")
+    public String test() {
+        productService.save(new Product("iPhone A", new BigDecimal("200")));
+        productService.save(new Product("iPhone B", new BigDecimal("500")));
+        productService.save(new Product("iPhone C", new BigDecimal("100")));
+        productService.save(new Product("iPhone D", new BigDecimal("150")));
+        productService.save(new Product("iPhone E", new BigDecimal("900")));
+        productService.save(new Product("iPhone F", new BigDecimal("1000")));
+        productService.save(new Product("iPhone G", new BigDecimal("300")));
+        productService.save(new Product("iPhone H", new BigDecimal("999")));
+        productService.save(new Product("iPhone I", new BigDecimal("332")));
+        productService.save(new Product("iPhone J", new BigDecimal("450")));
+        return "Done";
+    }
```

Comment:
I don't like creating endpoints for initialization purposes.
Not critical but it's redundant code in "production" class.
At least some obvious separate util with @PostConstruct method is already better